package.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash -ex
-export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11'
+
+export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11.4.0-04381b7'
 
 docker run --rm $DEBIFY_IMAGE config script > docker-debify
 chmod +x docker-debify
 
+# Pulling private image explicitly and manually to local repository
+# We don't pass registry credentials to debify, remove this when registry.tld/ruby-fips-base-image-phusion:1.0.0 become public.
```

Comment:
@aloncarmel111 do we have an open issue to remove this? When will this happen?

Reply:
we are not planning to push this commit, it's just to get green build.
we will remove this commit and then push this PR only  when conjur-base-image publish to dockerhub